src/store.ts

```diff
@@ -60,6 +82,18 @@ const store: Store<C4CState, C4CAction> = createStore<
   C4CAction,
   {},
   {}
->(reducers, initialStoreState, enhancer);
+>(reducers, preloadedState || initialStoreState, enhancer);
+
+store.subscribe(
+  throttle(() => {
```

Comment:
Not blocking at all, but could you explain throttle here? Just curious

Reply:
Yeah, just limits the function to running every 10 seconds :)